CMakeLists.txt

```diff
@@ -107,3 +107,30 @@ install(
     DESTINATION
     ${CMAKE_INSTALL_LIBDIR}/pkgconfig
 )
+
+option(DOCUMENTATION "Build the documentation" OFF)
```

Comment:
Could we call this `BUILD_DOCS` instead? It seems to be the name that most CMake projects use as a convetion, and some build systems automatically pass `-DBUILD_DOCS=` to CMake automatically ([Buildroot, for example](https://git.busybox.net/buildroot/tree/package/pkg-cmake.mk?id=89964814cbbb9aeb12b935da6c3268cff5a2a80d#n144)).

Reply:
Done, I do not know anything about CMake and the conventions around it, thanks for letting me know! :-)